crates/dotrix_ui/src/renderer/overlay.rs

```diff
@@ -0,0 +1,30 @@
+use wgpu::util::DeviceExt;
+
+use dotrix::{
```

Comment:
There I have problem. Rust analyzer shows errors with unresolved imports. Doesn't show dotrix crate in code completion.
But it will compile using cargo run

Reply:
you need to enable loadOutDirs setting for rust analyzer